### DISCUSSION TITLE
can it also fetch transcript of youtube video ?

### DISCUSSION THREAD

**Discussion by Greatz08** [LuanRT/YouTube.js#641] [General]
I know there is one pip package capable of this task but I wanted to know if this project is capable to capture transcript of youtube video or not

**Comment by jassonbla** [comment]
Hi @HakaishinShwet 

Have you ever used [this function](https://github.com/LuanRT/YouTube.js/blob/5ff30e12ded29afff5a758cbe96ac3418e6d20aa/src/core/mixins/MediaInfo.ts#L170) before?
I've checked the transcript implement, and we can use this function.


```typescript
const youtube = await Innertube.create({
  location: "US",
  lang: "en",
});

const video = await youtube.getInfo("video_id");
const transcript = await video.getTranscript();
```

In my environment, It's working well.

**Reply by jassonbla** [reply]
also you can read [the following discussions](https://github.com/LuanRT/YouTube.js/discussions/513).
